.travis.yml

```diff
@@ -11,12 +11,18 @@ addons:
     packages:
       - shellcheck
 
+env:
+  - TEST_RUN="./tests/test-kubernetes.sh"
```

Comment:
While putting this pull request together, should we add our other test script(s) at the same time?